### ISSUE TITLE
Magic DRC rule file is invalid!

### ISSUE THREAD

**Issue by d-m-bailey** [efabless/mpw_precheck#135]
Current rule file results in (spacing added)
```
Magic 8.3 revision 315 - Compiled on Fri Jul  1 17:37:25 UTC 2022.
Starting magic under Tcl interpreter
Using the terminal as the console.
Using NULL graphics device.
Processing system .magicrc file

Error parsing "/home/kanobailey/mpw_precheck/checks/tech-files/sky130A.magicrc": invalid command name "EF"
Bad local startup file "/home/kanobailey/mpw_precheck/checks/tech-files/sky130A.magicrc", continuing without.

Loading "/home/kanobailey/mpw_precheck/checks/drc_checks/magic/magic_drc_check.tcl" from command line.
Don't know how to read GDS-II:
Nothing in "cifinput" section of tech file.
[INFO]: Loading user_analog_project_wrapper

File user_analog_project_wrapper.mag couldn't be read
No such file or directory
Creating new cell
"drc(full)" is not one of the DRC styles Magic knows.
The current style is "default".
The DRC styles are: default.
No errors found.
[INFO]: COUNT: 0
[INFO]: Should be divided by 3 or 4
[INFO]: DRC Checking DONE (/home/kanobailey/mpw-7/caravel_user_project_analog/precheck_results/03_AUG_2022___00_44_55/outputs/reports/magic_drc_check.drc.report)
[INFO]: Saving mag view with DRC errors(/home/kanobailey/mpw-7/caravel_user_project_analog/precheck_results/03_AUG_2022___00_44_55/outputs/user_analog_project_wrapper.magic.drc.mag)
[INFO]: Saved
```
